.vscode/settings.json

```
@@ -152,11 +135,9 @@
   // Whether the extension contributes a lint task to lint a whole workspace.
   "eslint.lintTask.enable": true,
 
+  "eslint.workingDirectories": ["./"],
+
   // Avoids having to globally install ESLint.
   // TODO(DerekNonGeneric): See about switching to pnpm!
```

```suggestion
  // Avoids having to globally install ESLint.
  // TODO(DerekNonGeneric): See about switching to pnpm!
```

Actually, maybe not in this particular repo&hellip;